### DISCUSSION TITLE
Forcer forwards to pass events to receivers.

### DISCUSSION THREAD

**Discussion by alveare1** [Security-Onion-Solutions/securityonion#12830] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8 to 16 depending on node

### RAM

16 or 32gb

### Storage for /

512

### Storage for /nsm

512

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

https://github.com/Security-Onion-Solutions/securityonion/discussions/12777

I have changed the setting and have built a new distributed test environment but I am not able to get the events to start flowing while this setting is changed. 

I don't see the logs even being read. Everything is looks good and I am not seeing any errors in the logs.
I have verified paths and it shows everything looks good but it isn't opening and reading the logs

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
> I don't see the logs even being read.

Can you explain this further? 

With that setting applied when you go to elastic fleet and click on the 'Settings' tab, under 'Outputs' what do you have listed?

**Comment by alveare1** [comment]
I found the issue. When enabling this make sure all DNS issues are resolved. Was missing my search nodes in DNS. Cleared up once resolved.
